test/TesterInternal/packages.config

```diff
@@ -12,7 +12,6 @@
   <package id="WindowsAzure.Storage" version="5.0.2" targetFramework="net451" />
   <package id="xunit" version="2.1.0" targetFramework="net451" />
   <package id="xunit.abstractions" version="2.0.0" targetFramework="net451" />
-  <package id="xunit.assert" version="2.1.0" targetFramework="net451" />
```

Comment:
this is probably incorrect. We still need xunit.assert. In fact, the TesterInternal.csproj file is still referencing that dll from the nuget package.